vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -164,32 +127,39 @@ Promote the admin user to a full `conference-admin` to reinstate the ability to
 5. Confirm the delete icon is visible 
 6. Confirm a Conference can be successfully deleted from the list
 
-## Notes
-### Realm Roles versus Client Authorities
-This tutorial utilizes authorities. In Keycloak, authorities are roles mapped to a user for a specific client. It is possible to assign higher-level Realm Roles directly to users, e.g. `ROLE_ADMIN`, but this can result in collisions between applications using the same roles.
+### Step 10. Configure the roles in `entando.json`
```

Comment:
prefer this reference to `entando.json` which is not prefaced by "the"; minor but technically correct to *not* use "the"